src/libcore/ops.rs

```diff
@@ -235,6 +235,44 @@ pub trait Drop {
 /// }
 /// ```
 ///
+/// Here is an example of the same `Point` struct implementing the `Add` trait
+/// using generics.
+///
+/// ```
+/// use std::ops::Add;
+///
+/// #[derive(Debug)]
```

Comment:
I don't see any usage of the debug in the following code. Did you forget to remove it?